infra/base-images/base-runner/run_fuzzer

```diff
@@ -129,5 +138,9 @@ echo $CMD_LINE
 # Unset OUT so the fuzz target can't rely on it.
 unset OUT
 
+if [ ! -z $DEBUGGER ]; then
+  CMD_LINE="$DEBUGER $CMD_LINE"
```

Comment:
s/DEBUGER/DEBUGGER/

Reply:
wow, thanks a lot!

Reply:
spent some time trying to understand why my testing didn't catch this... realized that I didn't test `base-runner-debug` image (it's important as use of debugger happens there, not in `base-runner`), and now finally caught one more problem (double quotes are needed for this `! -z` check). thanks again!